app/src/components/Header/styled.ts

```diff
@@ -0,0 +1,62 @@
+import styled, { css } from 'styled-components'
+
+export const Navbar = styled.nav`
+  ${({ theme }) => css`
+    background-color: ${theme.color.background.page.white};
+    z-index: 5;
+    color: ${theme.color.text.primary};
+    top: 0;
+    position: fixed;
+    width: 100%;
+  `}
+`
+export const MenuContainer = styled.div.attrs({
+  className:
+    'container mx-auto flex flex-wrap p-5 top-0 md:flex-row items-center'
+})`
+  ${({ theme }) => css``}
```

Comment:
If you don’t have css to add to the styled-component, just use the element and put class.